packages/freerdp.rb

```diff
@@ -2,29 +2,53 @@
 
 class Freerdp < Package
   description 'FreeRDP is a free implementation of the Remote Desktop Protocol.'
-  homepage 'http://www.freerdp.com/'
-  version '2.0.0-rc4'
+  homepage 'https://www.freerdp.com/'
+  version '2.0.0-rc4-1'
   source_url 'https://github.com/FreeRDP/FreeRDP/archive/2.0.0-rc4.tar.gz'
   source_sha256 '3406f3bfab63f81c1533029a5bf73949ff60f22f6e155c5a08005b8b8afe6d49'
 
-  depends_on 'ffmpeg'
-  depends_on 'gst_plugins_base'
   depends_on 'cups'
+  depends_on 'mesa'
   depends_on 'xmlto'
-  depends_on 'libxslt'
   depends_on 'ninja'
   depends_on 'xprop'
-  depends_on 'sommelier'
+  depends_on 'ffmpeg'
+  depends_on 'libsoxr'
+  depends_on 'xorg_lib'
```

Comment:
libsoxr and xorg_lib come along for the ride with ffmpeg so not needed here.

Reply:
All right. In case you're wondering why I removed the `sommelier` dependency, it's because I feel that users should have a choice on what X server they use.